CHANGES.md

```diff
@@ -51,6 +51,15 @@
 -   fix passing gdal creation parameters in `write_dataframe` (#62)
 -   fix passing kwargs to GDAL in `write_dataframe` (#67)
 
+### Contributors
+
+People with a “+” by their names contributed a patch for the first time.
```

Comment:
FYI, this is actually meant to be generated using https://github.com/pandas-dev/pandas/blob/v1.4.2/doc/sphinxext/contributors.py.

Reply:
@brendan-ward I do this for geopandas as well (although I only seem to include it in the github release notes, and not in the changelog file (I assume for length purposes, but that's not (yet) an issue here): https://github.com/geopandas/geopandas/releases/tag/v0.10.0 
I have a script locally to create that (based on the pandas one), will share that.